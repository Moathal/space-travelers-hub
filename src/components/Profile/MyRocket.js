import React from 'react';

function MyRocket({ name }) {
  return (
    <h4>{name}</h4>
  );
}

export default MyRocket;
