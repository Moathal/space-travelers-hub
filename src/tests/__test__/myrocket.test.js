import renderer from 'react-test-renderer';
import MyRocket from '../../components/Profile/MyRocket';

it('renders MyRocket correctly', () => {
  const tree = renderer.create(<MyRocket />).toJSON();
  expect(tree).toMatchSnapshot();
});
