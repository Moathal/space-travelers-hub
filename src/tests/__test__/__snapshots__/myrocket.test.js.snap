// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders MyRocket correctly 1`] = `<h4 />`;
